apps/explorer/src/components/top-packages/TopPackagesTable.tsx

```diff
@@ -22,7 +30,11 @@ export function TopPackagesTable({ data, isLoading }: TopPackagesTableProps) {
 				),
 				function: <Text variant="bodySmall/medium">{item.function}</Text>,
 				package: <ObjectLink objectId={item.package} />,
-				count: <Text variant="bodySmall/medium">{count}</Text>,
+				count: (
+					<div className="text-right">
+						<Text variant="subtitle/semibold">{Number(count).toLocaleString()}</Text>
```

Comment:
@mamos-mysten Updated commas

Reply:
according to Figma this should be `pBody/medium`